Jenkinsfile

```diff
@@ -30,7 +30,7 @@ pipeline {
       }
       steps {
         script {
-          def dockerImage = docker.build 'linagora/esn-frontend-inbox'
+          def dockerImage = docker.build('linagora/esn-frontend-inbox', '--pull --no-cache .')
```

Comment:
Should we add it for the release tag step too?

Reply:
you are right i didnt see the other line